backend/src/main/java/com/bakdata/conquery/models/query/ManagedQuery.java

```diff
@@ -105,16 +104,16 @@ protected void doInitExecutable(@NonNull DatasetRegistry namespaces, ConqueryCon
 		this.namespace = namespaces.get(getDataset());
 		this.involvedWorkers = namespace.getWorkers().size();
 		query.resolve(new QueryResolveContext(getDataset(), namespaces));
-		if(label == null) {
-			label = makeAutoLabel(namespaces);
+		if (label == null) {
+			label = makeAutoLabel(namespaces, new PrintSettings(true, Locale.ROOT,namespaces));
```

Comment:
Auch nicht ROOT nehmen

Reply:
Warum brauchst du noch mal die Locale hier? Du willst doch nciht AND und OR in den Execution Namen packen oder?

Reply:
Falls ja, wir hatten uns eigentlich darauf geeinigt keine logischen Verknüpfungen im Namen abzubilden.

Reply:
Ne ist nicht für den Namen an sich, aber das mögliche lokalisieren aller lables

Reply:
Gut lassen wir es erstmal so,  es hat ja keine Auswirkung